### PR TITLE
[CI] Separate android CI and merge manual execute action

### PR DESCRIPTION
**Pull request by Atralupus**
- I noticed that .github/workflows/build-via-workflow-dispatch.yaml contains a significant amount of duplicated code, so I've merged them.
- With the introduction of the Android CI, the build CI has accumulated many conditional statements. It's now time to separate these jobs.

### CODE REVIEW

File: .github/workflows/android-build-and-release.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        run: |
+          tr -d "\r" < .github/bin/install-bundletool.sh > .github/bin/install-bundletool-trimed.sh
+          bash .github/bin/install-bundletool-trimed.sh /tmp
+
+      - name: Setup Keystore and Passwords
+        run: |
+          echo "${{ secrets.ANDROID_KEYSTORE_BASE64 }}" | base64 -d > /tmp/9c-signing-keystore.keystore
+          echo "${{ secrets.ANDROID_KEYSTORE_PASS }}" > /tmp/keystore.pwd
+          echo "${{ secrets.ANDROID_KEY_ALIAS_NAME }}" > /tmp/keyalias.name
+          echo "${{ secrets.ANDROID_KEY_ALIAS_PASS }}" > /tmp/key.pwd
```
Comment (boscohyun) [inline .github/workflows/android-build-and-release.yml:93]: Could you rename it like below?

```suggestion
          echo "${{ secrets.ANDROID_KEY_ALIAS_PASS }}" > /tmp/keyalias.pwd
```

File: .github/workflows/android-build-and-release.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Move Folder
+        run: |
+          sudo mv $GITHUB_WORKSPACE/build /tmp/player
+
+      - name: Convert AAB to APKS
+        run: |
+          sudo java -jar /tmp/bundletool-all-*.jar build-apks --bundle="/tmp/player/Android/9c.aab" --output="/tmp/player/Android/result.apks" --ks="/tmp/9c-signing-keystore.keystore" --ks-pass=file:/tmp/keystore.pwd --ks-key-alias=$(cat /tmp/keyalias.name) --key-pass=file:/tmp/key.pwd --mode=universal --local-testing
```
Comment (boscohyun) [inline .github/workflows/android-build-and-release.yml:107]:
```suggestion
          sudo java -jar /tmp/bundletool-all-*.jar build-apks --bundle="/tmp/player/Android/9c.aab" --output="/tmp/player/Android/result.apks" --ks="/tmp/9c-signing-keystore.keystore" --ks-pass=file:/tmp/keystore.pwd --ks-key-alias=$(cat /tmp/keyalias.name) --key-pass=file:/tmp/keyalias.pwd --mode=universal --local-testing
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -5,14 +5,19 @@ on:
...
   pull_request:
     types: [ready_for_review, opened, reopened, auto_merge_enabled]
+  workflow_dispatch:
+    inputs:
+      clo:
+        description: "Assets/StreamingAssets/clo.json"
+        required: false
+        type: string
+        default: ""
+
```
Comment (boscohyun) [inline .github/workflows/build.yml:20]: This inputs are not used.😅

**Review by boscohyun — COMMENTED** [review]
❤️

File: .github/workflows/android-build-and-release.yml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+          path: ${{ matrix.projectPath }}/Library
+          key: Library-${{ matrix.projectPath }}-ubuntu-${{ matrix.targetPlatform }}
+          restore-keys: |
+            Library-${{ matrix.projectPath }}-ubuntu-
+            Library-
+
+      - name: Copy clo.json
+        run: |
+          if [ -n "${{ github.event.inputs.clo }}" ]; then
+            echo ${{ github.event.inputs.clo }} > nekoyume/Assets/StreamingAssets/clo.json
```
[the remark this answers is not in the document]
Comment (Atralupus) [reply .github/workflows/android-build-and-release.yml:62]: Cool

File: .github/workflows/android-build-and-release.yml (hunk 2 above)
Comment (boscohyun) [inline .github/workflows/android-build-and-release.yml:107]: It seems needs to change.(#2896)
```suggestion
          sudo java -jar /tmp/bundletool-all-*.jar build-apks --bundle="/tmp/player/Android/NineChronicles.aab" --output="/tmp/player/Android/result.apks" --ks="/tmp/9c-signing-keystore.keystore" --ks-pass=file:/tmp/keystore.pwd --ks-key-alias=$(cat /tmp/keyalias.name) --key-pass=file:/tmp/key.pwd --mode=universal --local-testing
```

File: .github/workflows/android-build-and-release.yml (hunk 1 above)
Comment (Atralupus) [reply .github/workflows/android-build-and-release.yml]: Good. I'll apply this

File: .github/workflows/build.yml (hunk 3 above)
Comment (Atralupus) [reply .github/workflows/build.yml]: Done!

File: .github/workflows/android-build-and-release.yml (hunk 2 above)
Comment (Atralupus) [reply .github/workflows/android-build-and-release.yml]: yes... https://github.com/planetarium/NineChronicles/actions/runs/5885598414/job/15962264697, you're right

**Review by boscohyun — APPROVED** [review]

**Comment by Atralupus** [comment]
https://github.com/planetarium/NineChronicles/actions/runs/5885775691/job/15962695567
test done 🚀

**Pull request merged**